src/views/partials/nav.njk

```diff
@@ -1,31 +1,14 @@
-{% set sections = [
-    {
-        pages: [
-            {
-                url: "/",
-                title: "Getting started"
-            }
-        ]
-    }, {
-        title: "Styles",
-        pages: collections.styles | eleventyNavigation
-    }, {
-        title: "Components",
-        pages: collections.components | eleventyNavigation
-    }
-] %}
-
-<nav>
+<nav class="hidden md:block md:py-6">
     <ul>
         {% for section in sections %}
             <li>
                 {% if section.title %}
                     <h2 class="font-normal text-base text-grey-2 mb-2">{{section.title}}</h2>
                 {% endif %}
-                <ul class="mb-4">
+                <ul class="md:mb-4">
                     {% for post in section.pages %}
                         {% if page.url == post.url %}
```

Comment:
I know not being introduced here but the tabbing in this file is a little funky, else and endif don't align.

Reply:
Yeah this is the fault of Prettier (or whatever it is formatting .njk files in my vscode) - I'll take a look at something more robust.

Reply:
Looking at it, I think the if/else combined with `<li>` is what's confusing it.